src/components/table/index.tsx

```diff
@@ -0,0 +1,82 @@
+"use client";
+
+import { Table, TableHead, TableHeaderCell, TableBody, TableRow, TableCell } from "@tremor/react"
+import { useEffect, useState } from "react";
+import type { ColumnDefinition, TableState } from "@/types/tableTypes";
+
+export default function CatchmentTable({

```

Comment:
Ještě bych to změnil na generic props - Promise<T[]>. Viz https://medium.com/edonec/creating-a-generic-component-with-react-typescript-2c17f8c4386e

Reply:
Jo, to je určitě lepší. Tohle byl jen takový první nástřel. Upravím.